File: cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float dollars;
    int count = 0;
    int quarters = 25;
    int dimes = 10;
    int nickels = 5;
    int pennies = 1;
    do
    {
        dollars = get_float("Change owed: ");
        
    }
    while (dollars < 0);
    int cents = round(dollars * 100);

    while (cents >= quarters)
    {
        cents = cents - quarters;
        count++;
    }
    while (cents >= dimes)
    {
        cents = cents - dimes;
        count++;
    }
    while (cents >= nickels)
    {
        cents = cents - nickels;
        count++;
    }
    while (cents >= pennies)
    {
        cents = cents - pennies;
        count++;
    }
   
    printf("%i\n", count);
}

File: cash.py
while True:
    try:
        no = float(input("Changed owed: "))
        if no >= 0:
            break
    except:
        continue
no = round(no * 100)
count = 0
for i in [25, 10, 5, 1]:
    while no >= i:
        count = count + no//i
        no = no %i
    if no == 0:
        break
print(count)